docs/integrations/Rezolve AI.md

```diff
@@ -0,0 +1,45 @@
+# Rezolve AI Integration
+
+Rezolve’s AI platform specialises in conversational commerce, offering product-based recommendations and contextual data for a richer user experience. Once you set up this integration, your Bluedot-enabled app can request real-time commerce intelligence, stream dynamic responses, and incorporate product data into chat sessions.
+
+---
+
+## Prerequisites
+
+1. **Rezolve AI Account**: Make sure you have an active Rezolve account and have access to your API Key and Endpoint. You can learn more at [Rezolve’s website](https://rezolve.com/).
```

Comment:
Need to replace `Rezolve’s website` with `the Rezolve website` - as given we are owned by Rezolve, this website is technically "Rezolve's website".